test_conformance/extensions/cl_khr_command_buffer/cl_khr_command_buffer_mutable_dispatch/mutable_command_basic.h

```diff
@@ -103,6 +103,7 @@ struct BasicMutableCommandBufferTest : BasicCommandBufferTest
     clUpdateMutableCommandsKHR_fn clUpdateMutableCommandsKHR = nullptr;
 
     const char* kernelString = "__kernel void empty() {}";
+    const size_t global_work_size = 4 * sizeof(cl_int);
```

Comment:
redeclaration of `global_work_size` below

Reply:
I don't understand.